keps/sig-node/1797-configure-fqdn-as-hostname-for-pods/README.md

```diff
@@ -626,17 +628,19 @@ _This section must be completed when targeting beta graduation to a release._
 
 _This section must be completed when targeting beta graduation to a release._
 
-TODO
 
 * **How can an operator determine if the feature is in use by workloads?**
   Ideally, this should be a metrics. Operations against Kubernetes API (e.g.
   checking if there are objects with field X set) may be last resort. Avoid
   logs or events for this purpose.
 
+  For a pod to be using this feature it needs to define the PodSpec fields
+  `subDomain` and `setHostnameAsFQDN`.
+
 * **What are the SLIs (Service Level Indicators) an operator can use to
   determine the health of the service?**
-  - [ ] Metrics
-    - Metric name:
+  - [X] Metrics
+    - Metric name: Abnormal increase in `run_podsandbox_errors_total` might be related to this feature. We should check if the feature gate is enabled and pods are using it.
```

Comment:
Let's put only the metric name here.

And add "- Comment: " below with the explanation.

Reply:
ok